src/main/java/polyglot/CommandLineArgs.java

```diff
@@ -47,6 +47,9 @@
   @Option(name = "--oauth2_token_endpoint")
   private String oauth2TokenEndpoint;
 
+  @Option(name = "--oauth2_access_token")
```

Comment:
Please don't force people to pass plaintext credentials, but point to a file (similar to refresh token). Also please make sure there's a metaVar (which goes into usage).